package.json

```diff
@@ -88,7 +88,7 @@
     "typescript": "^3.4.5",
     "uglify-js": "^3.5.8",
     "underscore-template-loader": "^1.0.0",
-    "webdriver-manager": "^13.0.0",
+    "webdriver-manager": "12.1.7",
```

Comment:
Is this required?

Reply:
`13.0.0` was a beta version of webdriver-manager and the latest version if `12.1.7` (see [here](https://www.npmjs.com/package/webdriver-manager)). I found certain bugs related to fetching and using the correct chromedriver version with the beta version and that's why I have changed it.

Reply:
Can you change it to `^12.1.7`? Since `13.0.0` is major version it won't get updated to that version, it will just preserve latest `12.x.y` version.

Reply:
Ah yes, I'll do that in my next commit. Thanks!

Reply:
Done.